### PR TITLE
installation section added

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

**Comment by YogiSoni-lazy** [comment]
Last section is WIP, let me know what you think overall.

File: package-lock.json [hunk 1]
```diff
@@ -4,11 +4,12 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "eap-admin1",
       "devDependencies": {
         "@antora/cli": "3.1.3",
         "@antora/site-generator": "3.1.3",
         "http-server": "^14.1.1",
-        "watch": "^1.0.2"
+        "watch": "^0.13.0"
```
Comment (rsriniva) [inline package-lock.json:12]: why downgrade version?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "@antora/cli": "3.1.3",
     "@antora/site-generator": "3.1.3",
     "http-server": "^14.1.1",
-    "watch": "^1.0.2"
+    "watch": "^0.13.0"
```
Comment (rsriniva) [inline package.json:6]: why the downgrade?

File: modules/install/pages/install.adoc [hunk 3]
```diff
@@ -1,71 +1,538 @@
-= Installing JBoss EAP
+== Red Hat JBoss Enterprise Application Platform 8 Installation Methods
 
-Brief blurb about how JBoss EAP offers multiple ways to install...
+You can install JBoss EAP 8.0 by using one of the following installation methods:
 
-== Pre-requisites
-
-Talk about what pre-requisites are needed:
+* *JBoss EAP using the Zip file*
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: Remove JBoss EAP using prefix in all these bullets

Comment (rsriniva) [reply modules/install/pages/install.adoc]: On first reading, zip file and archive look the same? is archive packaged as tar.gz?

File: modules/install/pages/install.adoc [hunk 4]
```diff
@@ -1,71 +1,538 @@
...
+
+* JBoss EAP 8 is available as an RPM package for users with a valid subscription to the JBoss Enterprise Application Platform sub-channel, in the JBoss Enterprise Application Platform group.
+* The yum command downloads all the RPMs to the host OS, and automatically installs all the files needed to start and manage JBoss EAP.
+* The RPM installation method is only valid for Red Hat Enterprise Linux (RHEL) operating system and not for Windows server.
+* This method is preferred when you want JBoss EAP service files to be installed for the RHEL operating system.
 
 == Installing JBoss EAP using the JAR installer
 
-Brief overview only. Details in lab section below....
+* This method can also be called the GUI installer method.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: Add a sentence that this provides an "interactive" graphical approach where you answer a series of questions

File: modules/install/pages/install.adoc (hunk 3 above)
Comment (rsriniva) [reply modules/install/pages/install.adoc]: Can we merge the first two options into one and call it a common name - like archive files. zip for windows and tar.gz for UNIX like platforms?

File: modules/install/pages/install.adoc [hunk 5]
```diff
@@ -1,71 +1,538 @@
...
+* Due to its flexible nature, the installer can install JBoss EAP by using a response file, with all the options passed as an XML file.
 
-== Installing JBoss EAP using Installation Manager
+== Pre-requisites
 
-Brief overview only. Details in lab section below....
+* You have access to the internet.
+* You have reviewed https://access.redhat.com/articles/6961381[JBoss EAP 8.0 supported configurations].
+* You have created an account on the https://access.redhat.com/[Red Hat Customer Portal] to download JBoss EAP files.
+* You are operating a RHEL 9 server equipped with a graphical user interface (GUI) package.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: This should not be mandatory. This is mandatory only if you need to run the graphical JAR installer in GUI mode.

File: modules/install/pages/install.adoc [hunk 6]
```diff
@@ -1,71 +1,538 @@
...
+* You have installed https://access.redhat.com/articles/6961381[Red Hat supported OpenJDK 17] on RHEL 9 server.
+* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin).
+* Recommended to download all installation file under /opt partition.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:42]: not mandatory. They can download wherever. They install into /opt

File: modules/install/pages/install.adoc [hunk 7]
```diff
@@ -1,71 +1,538 @@
...
+* Recommended to ensure there is enough space under /opt partition (minimum 800MB).
 
-== Installing JBoss EAP using other methods....
+== (Optional) Pre-configured RHEL9 settings using Vagrant
 
-Brief overview only. Details in lab section below....
+* You can refer https://github.com/RedHatQuickCourses/eap-qc-apps[this eap8 vagrant settings] to configure the RHEL 9 server to get started.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: This does not link to the vagrant folder. I left a comment in the other repo to rename the folder and move it under the admin1 folder. Please ensure both sides of the reference match

File: modules/install/pages/install.adoc [hunk 8]
```diff
@@ -1,71 +1,538 @@
...
 
-== Starting JBoss EAP
+== Lab 1: Installing JBoss EAP using the Zip file
 
-Provide a brief explanation of different flags for the _standalone.sh_ script, especially *-b* flag....default is bound only to localhost (127.0.0.1) for security.
+*Outcome*: In this lab, you download `jboss-eap-8.0.0-installation-manager.zip` file and install JBoss EAP as a standalone server that can be access at `http://localhost:8080`.
 
-NOTE: Mention that the installation can be automated using a response file (kinda like kickstart), and that this will be covered in the second course in the path (_Advanced JBoss EAP Administration_)
+* Take GUI access of RHEL 9 server.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:53]: Re-word this sentence. What does take mean in this context? Is graphical login really required? Can we get away with Log in to the RHEL 9 VM using SSH?

File: modules/install/pages/install.adoc [hunk 9]
```diff
@@ -1,71 +1,538 @@
...
 
-NOTE: Mention that the installation can be automated using _Ansible Middleware_, and that this will be covered in the second course in the path (_Advanced JBoss EAP Administration_)
+* Download the `jboss-eap-8.0.0-installation-manager.zip` by following below steps:
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: I think there is no need to download from the server. Can we keep the instructions generic and say download on your local machine and then copy it to the server? maybe using scp?

File: modules/install/pages/install.adoc [hunk 10]
```diff
@@ -1,71 +1,538 @@
...
+[NOTE]
+--
+The jboss-eap-installation-manager is also supported on Microsoft Windows. To use this script on a Windows machine, replace the `.sh` with `.bat` in your script.
+--
+
+* Shutdown the JBoss EAP 8 server by pressing `Ctrl+C` in the terminal window where you started JBoss EAP 8.
+
+* To uninstall the JBoss EAP 8 server, simple remove the `/opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001` directory.
+
+== Lab 2: Installing JBoss EAP using archive method
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: I am totally confused. SHould the first lab and option be renamed to "Installing JBoss EAP using the EAP Installation Manager"?

File: modules/install/pages/install.adoc [hunk 11]
```diff
@@ -1,71 +1,538 @@
...
+
+* Switch to the `bin` directory present under the uncompressed `jboss-eap-8.0.0.zip` file.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server opt]$ cd jboss-eap-8.0/bin/
+[vagrant@server bin]$
+----
+
+* Install JBoss EAP by executing the `./jboss-eap-installation-manager.sh` script as a sudo user.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: why is sudo needed? I thought we agreed to run EAP as a non-root normal user called jboss? are you missing chmod commands somewhere in the hands on lab?

File: modules/install/pages/install.adoc [hunk 12]
```diff
@@ -1,71 +1,538 @@
...
+JBoss examples installed.
+Server created in /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8
+Operation completed in 541.33 seconds.
+----
+
+* Start the JBoss EAP standalone server using the start script present under `/opt/jboss-eap-8.0/bin/eap-8/bin/standalone.sh`.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo /opt/jboss-eap-8.0/bin/eap-8/bin/standalone.sh
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: not a good idea to run as root

File: modules/install/pages/install.adoc [hunk 13]
```diff
@@ -1,71 +1,538 @@
...
+* Install JBoss EAP 8.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo dnf groupinstall jboss-eap8 -y
+----
+
+NOTE: The default `EAP_HOME` path for the RPM installation is `/opt/rh/eap8/root/usr/share/wildfly`.
+
+* Execute the JBoss EAP standalone server script present at `EAP_HOME/bin/standalone.sh` as a sudo user.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:172]: should not be run as root. Anyway command below is missing sudo. Should be run as jboss user

File: modules/install/pages/install.adoc [hunk 14]
```diff
@@ -1,71 +1,538 @@
...
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo dnf groupremove jboss-eap8 -y
+----
+
+== Lab 4: Installing JBoss EAP using GUI method and as a RHEL service
+
+*Outcome*: In this lab, you download `jboss-eap-8.0.0-installer.jar` file and install JBoss EAP as a standalone server that can be access at `http://localhost:8080`. You further configure JBoss EAP to run as a service in RHEL to enable the JBoss EAP service to start automatically when the RHEL server starts.
+
+* Take GUI access of RHEL 9 server.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: Reword as commented earlier

File: modules/install/pages/install.adoc [hunk 15]
```diff
@@ -1,71 +1,538 @@
...
+NOTE: The default `EAP_HOME` path for the RPM installation is `/opt/rh/eap8/root/usr/share/wildfly`.
+
+* Execute the JBoss EAP standalone server script present at `EAP_HOME/bin/standalone.sh` as a sudo user.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ /bin/sh /opt/rh/eap8/root/usr/share/wildfly/bin/standalone.sh -c standalone.xml -b 0.0.0.0
+----
+
+* Navigate to the JBoss EAP console at `http://localhost:8080` using firefox browser and confirm JBoss EAP is running as shown in the below screenshot.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: You have repeated the localhost:8080 home page thrice. Just do it once and mention in the step that "You should see the home page".

File: modules/install/pages/install.adoc [hunk 16]
```diff
@@ -1,71 +1,538 @@
...
++
+image::ui.png[align="center"]
+
+== Lab 5: Uninstalling JBoss EAP service on RHEL
+
+*Prerequisites*:
+You have JBoss EAP installed, or your have followed Lab 4
+
+*Outcome*: You remove an instance of JBoss EAP and any services associated with it.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc]: These steps only remove the service. Rename this section as "Uninstalling JBoss EAP" and then first show them how to uninstall JBoss EAP using the uninstallation GUI wizard followed by how to remove the service. FOr the other options write a sentence or two about simply deleting the folder under /opt - does the installation manager shell scriptn offer an option to uninstall just the like GUI JAR installer?

**Review by rsriniva — CHANGES_REQUESTED** [review]
Much better structure. Need to address some more comments before merging. Excellent work

File: modules/install/pages/install.adoc (hunk 3 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: done the correction

File: modules/install/pages/install.adoc (hunk 4 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: added the wordings

File: modules/install/pages/install.adoc (hunk 5 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: Removed these pre-requisites

File: modules/install/pages/install.adoc (hunk 12 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: done the correction

File: modules/install/pages/install.adoc (hunk 13 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:172]: done the correction

File: modules/install/pages/install.adoc (hunk 15 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: Done the correction

File: modules/install/pages/install.adoc (hunk 16 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc]: Added the section

**Review by rsriniva — APPROVED** [review]
I am approving this for now and merging. I'll create a small PR for some minor adjustments later on. Great work Yogita! This looks very comprehensive!

**Pull request merged**